cli/compose/loader/volume.go

```diff
@@ -70,6 +70,8 @@ func populateFieldFromBuffer(char rune, buffer []rune, volume *types.ServiceVolu
 		switch option {
 		case "ro":
 			volume.ReadOnly = true
+		case "rw":
```

Comment:
I think there are 2 concerns:

1. BC: Maintain backwards compatibility with Docker Compose. Anything that used to work with Docker Compose should work here. Of course you could argue that `docker stack` never supported some options, but that would still mean users who are upgarding / migrating from Docker Compose to `docker stack` would be caught by surprise. In which case, there needs to be clear documentation on what's supported and what's not, and ideally mentioned in the upgrade / migration guide.
2. Consistency: If the supported options are different from what's in `docker run`, we need to ask the question of why it should be that way. (I as a user certainly am confused as to why.) And if the decision is to support a different set of options, then it needs to be clearly documented.